canonicalwebteam/blog/django/urls.py

```diff
@@ -48,9 +49,10 @@ def to_url(self, value):
     path(r"/<yyyy:year>/<mm:month>/<dd:day>/<str:slug>", article_redirect),
     path(r"/<yyyy:year>/<mm:month>/<str:slug>", article_redirect),
     path(r"/<yyyy:year>/<str:slug>", article_redirect),
+    path(r"/<str:slug>", article, name="article"),
```

Comment:
If this is up here, won't it clobber `/feed` and `/latest-news`?

Reply:
It does indeed :) Fixed